__tests__/account.test.ts

```diff
@@ -167,7 +168,7 @@ describe('deploy and test Wallet', () => {
       expect(declareTx.class_hash).toBeDefined();
     });
 
-    test('Get the stark name of the account and account from stark name (using starknet.id)', async () => {
+    test.only('Get the stark name of the account and account from stark name (using starknet.id)', async () => {
```

Comment:
I think you need to remove `.only` else other tests will not run ?

Reply:
Yep my bad, fixed